src/Nethermind/Nethermind.State/Healing/HealingStorageTree.cs

```diff
@@ -65,11 +65,11 @@ public override void Set(ReadOnlySpan<byte> rawKey, byte[] value)
         }
     }
 
-    private bool Recover(in TreePath missingNodePath, in ValueHash256 hash, Hash256 fullPath)
+    private bool Recover(in TreePath missingNodePath, in Hash256 hash, Hash256 fullPath)
```

Comment:
No `in` with ref type; or is a pointer of a pointer for double pointer chase
```suggestion
    private bool Recover(in TreePath missingNodePath, Hash256 hash, Hash256 fullPath)
```